WalletWasabi/WabiSabi/WabiSabiCoordinator.cs

```diff
@@ -49,11 +55,23 @@ private void Arena_CoinJoinBroadcast(object? sender, Transaction e)
 		File.AppendAllLines(Parameters.CoinJoinIdStoreFilePath, new[] { e.GetHash().ToString() });
 	}
 
+	private void FeeRateStatStore_NewStat(object? sender, CoinJoinFeeRateStat feeRateStat)
+	{
+		if (!File.Exists(Parameters.CoinJoinFeeRateStatStoreFilePath))
```

Comment:
This if unnecessary. The content of the if `IoHelpers.EnsureContainingDirectoryExists(Parameters.CoinJoinFeeRateStatStoreFilePath);` does the check.

Reply:
Also this should happen in an initializer method, like a constructor.

Reply:
I would put the check on L60 in the constructor or maybe better in an initialization method and that's because file systems are similar to networking code in the sense that until you do an operation (file append here) you don't really know if it succeeds or not. 

I mean here we check on L60-L63 that a directory exists so that we can append a file, but it's not ruled out that on L64 the folder will be deleted by some third party. So this code drastically reduces the chance that L65 is called without existing folder but it's still possible.

Reply:
Fixed.